### PR TITLE
Use advanced terminal library

### LINKED ISSUE #57: Use advanced terminal library
Reimplement terminal-based version using [Questionary](https://pypi.org/project/questionary/).

### PR DESCRIPTION
**Pull request by IKostric**
Fixes #57

### CODE REVIEW

File: moviebot/agent/agent.py [hunk 1]
```diff
@@ -181,50 +206,65 @@ def continue_dialogue(self, user_utterance, user_options, user_fname=None):
 
         Args:
             user_utterance: The input received from the user
-            user_options: 
+            user_options:
```
Comment (NoB0) [inline moviebot/agent/agent.py:209]: Nit: missing description.

File: moviebot/run.py [hunk 2]
```diff
@@ -50,11 +54,12 @@ def arg_parse(args=None):
     """
     argv = args if args else sys.argv
     cfg_parser = None
-    print(argv)
     if len(argv) < 3:
-        print("WARNING: Configuration file is not provided.")
+        logger.warning("Configuration file is not provided.")
         config_file = r"config/moviebot_config.yaml"
```
Comment (NoB0) [inline moviebot/run.py:59]: Should this be a file variable?

File: moviebot/controller/controller_terminal.py [hunk 3]
```diff
@@ -15,9 +16,6 @@ def execute_agent(self) -> None:
         """Runs the conversational agent and executes the dialogue."""
         agent = None
         while True:
-            utterance = input("User: ")
-            user_utterance = UserUtterance({"text": utterance})
-
             if not agent or self.restart(user_utterance):
```
Comment (WerLaj) [inline moviebot/controller/controller_terminal.py:19]: user_utterance is not defined here now

**Review by WerLaj — APPROVED** [review]
LGTM

File: moviebot/controller/controller_terminal.py (hunk 3 above)
Comment (IKostric) [reply moviebot/controller/controller_terminal.py:19]: Fixed

**Pull request merged**